ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml

```diff
@@ -62,6 +62,7 @@ tests:
   - intranet
   cron: 6 4 16 1,3,5,7,9,11 *
   steps:
+    allow_skip_on_success: true
```

Comment:
It will skip all the post steps, am I right ? If true, `junit2reportportal` would also be skipped ?

Reply:
Nope. Individual `post` steps opt into being skipped by setting the `optional_on_success` field